### ISSUE TITLE
Can't display my server online

### ISSUE THREAD

**Issue by MrAnderon007** [kzahel/web-server-chrome#291]
Hi!
I just have 2 url for web server and i saw that some of you had a third one for sharing and access online. How can i have this third url ? Or can i share one of the others as well ? I can't find a way to give access to my server online
![image](https://user-images.githubusercontent.com/96294102/146540772-ac2ef262-4f49-42d9-a348-d1fbad1e71ec.png)
Thanks for your help

**Comment by stokito** [comment]
To have an access from the Internet you must have a public IP address so that any computer from outside can connect to it. Most users doesn't have it because their ISP (internet provider) uses a NAT. But some lucky users have the public IP but it's belong to a router in which the cable is plugged in. So public network betwen router and Internet is called WAN and local network called LAN. All devices and computers inside of LAN usually have IP addresses like 192.168.1.x.

Many routers have a UPnP IDG gateway which is a daemon (program) that can open and forward a port from WAN to LAN. A computer inside of LAN sends a UPnP request to the router and it forwards all network packets from one WAN port into the port of computer.

Basically this `Also on Internet` option just enables this feature.

If you do have a public IP but don't have a UPnP on the router then you can make a port forwarding manually.
You may find many tutorials for this.
